core/src/main/kotlin/com/playkids/onboarding/core/model/Item.kt

```diff
@@ -10,7 +12,19 @@ data class Item(
     val image: String,
     val currency: ItemCurrencies,
     val price: Int
-)
+){
+    companion object{
+        fun itemFactory(itemDTO: CreateItemDTO): Item{
```

Comment:
this is a secondary constructor, it can be implemented like [this](https://kotlinlang.org/docs/reference/classes.html)
```suggestion
    constructor(itemDTO: CreateItemDTO): this(...)
```